sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.0.0-preview.6 (2022-06-07)
 
-### Changed
+### Other Changes
```

Comment:
I assume if we do not make the change, CI fails?

Reply:
@ArcturusZhang Yes. I manually updated the Changelog because it is not complied with our guideline, and our changelog check script will fail in CI.